app/models/item.rb

```diff
@@ -25,7 +25,7 @@ def item_best_day
             .order(sales: :desc)
             .first.updated_at
   end
-
+  
```

Comment:
Not sure what's going on here.

Reply:
strange issue with the git diff it would seem!